src/components/autocompleter/autocompleter.jsx

```diff
@@ -0,0 +1,105 @@
+import PropTypes from 'prop-types';
+import React, { forwardRef, useEffect, useState } from 'react';
+import useFetch from '@bloodyaugust/use-fetch';
+import Select from '../select/select.jsx';
+import ListOption from '../list-option/list-option.jsx';
+import mockConstants from '../../mocks/mock-constants.js';
+
+const { API_ROOT } = mockConstants;
+
+const Autocompleter = forwardRef(function Autocompleter(props, ref) {
+  const { execute: fetch } = useFetch();
+  const [models, setModels] = useState(props.models);
+
+  useEffect(fetchModels, []);
+
+  function fetchModels(searchString) {
+    fetch(apiEndpoint(searchString))
+      .then((results) => {
+        setModels(results.json.results.map(resultInfo => results.json[resultInfo.key][resultInfo.id]));
```

Comment:
Reading this is pretty hard.

```suggestion
        setModels(respObj.json.results.map(result => respObj.json[result.key][result.id]));
```